new-frontiers-14/frontier-station-14
# Adds FSB Spirit poster and two EMS signs.

## About the PR



Adds a poster for the FSB Spirit for display and for incorporation into the dead drop library.
Adds two styles of "EMS" signs as alternatives to the MED sign, both for personal use and for anyone who thinks they don't look awful and would like to incorporate them.

Edit: Adds an addition poster from user incongruenciaastrologica on discprd. 

## Why / Balance


Advertisement/character for the Spirit.
Alternative choice sign for medical signs/posters who 

## How to test


Place posters. Check descriptions. Make sure they're part of the general smuggling list.

## Media

![posters](https://github.com/user-attachments/assets/246dcbcb-4b57-4dc3-b0c4-4c34fe0c0162)
![ss+(2024-09-28+at+11 19 58)](https://github.com/user-attachments/assets/536d70bd-4b3d-4c8e-a81e-4a939246a7f2)


## Requirements


## Breaking changes


**Changelog**



🆑 
- add: New poster for the FSB Spirit and two new EMS signs.
